File: app/src/main/java/com/lzm/lightLive/http/request/dy/DyConverter.java
package com.lzm.lightLive.http.request.dy;

import android.util.Log;

import com.lzm.lightLive.http.CustomGsonConvertFactory;

import java.io.IOException;

import okhttp3.ResponseBody;

public class DyConverter extends CustomGsonConvertFactory.ResponseConverter {

    @Override
    protected String convertResponse(ResponseBody value) throws IOException {
        String response = value.string();
        Log.e("TAG", response );
        return response;
    }
}

File: app/src/main/java/com/lzm/lightLive/http/request/dy/DyConverter.kt
package com.lzm.lightLive.http.request.dy

import android.util.Log
import com.lzm.lightLive.http.CustomGsonConvertFactory.ResponseConverter
import okhttp3.ResponseBody
import java.io.IOException

class DyConverter : ResponseConverter() {
    @Throws(IOException::class)
    override fun convertResponse(value: ResponseBody): String {
        val response = value.string()
        Log.e("TAG", response)
        return response
    }
}